ci/build_wheel.sh

```diff
@@ -8,14 +8,7 @@ source "$(dirname "$0")/install_latest_cuda_toolkit.sh"
 
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 
-# This is the version of the suffix with a preceding hyphen. It's used
-# everywhere except in the final wheel name.
-PACKAGE_CUDA_SUFFIX="-${RAPIDS_PY_CUDA_SUFFIX}"
-
-# Patch project metadata files to include the CUDA version suffix.
-sed -i "s/^name = \"pynvjitlink\"/name = \"pynvjitlink${PACKAGE_CUDA_SUFFIX}\"/g" pyproject.toml
-
-rapids-logger "Build wheel"
+rapids-logger "Build wheel"s
```

Comment:
```suggestion
rapids-logger "Build wheel"
```

Not sure how this got in there 😬